app/lib/rocketchat/services/restApi.ts

```diff
@@ -461,7 +461,7 @@ export const getListCannedResponse = ({ scope = '', departmentId = '', offset =
 	return sdk.get('canned-responses', params);
 };
 
-export const toggleBlockUser = (rid: string, blocked: string, block: boolean) => {
+export const toggleBlockUser = (rid: string, blocked: string, block: boolean): Promise<boolean> => {
```

Comment:
Does this call return true or false?
Isn't it an object or something?

Reply:
No! I was wondering exactly the same thing, but this returns only a boolean `true` if the request was completed for block and unblock the user! Or throw an error.